### PR TITLE
Feat/voip guide links

### PR DESCRIPTION
**Pull request by Steffy29**
Add links to user guides

ref: VoIP / TEL-5225

### CODE REVIEW

File: client/app/telecom/telephony/translations/Messages_fr_FR.xml [hunk 1]
```diff
@@ -112,6 +112,9 @@
    <translation id="telephony_phonebook_contact_delete_plural">Supprimer les {{ count }} contacts sélectionnés</translation>
    <translation id="telephony_phonebook_contact_delete_contacts_success">Suppression en cours…</translation>
 
+   <translation id="telephony_phonebook_contact_FAQ">Gestion des contacts - Guide</translation>
+
```
Comment (antleblanc) [inline client/app/telecom/telephony/translations/Messages_fr_FR.xml]: Extra empty line.

File: client/app/telecom/telephony/service/voicemail/management/translations/Messages_fr_FR.xml [hunk 2]
```diff
@@ -12,4 +12,5 @@
    <translation id="telephony_line_answer_voicemail_directories_caller" qtlid="303549">Appelant</translation>
    <translation id="telephony_line_answer_voicemail_directories_date" qtlid="24523">Date</translation>
    <translation id="telephony_line_answer_voicemail_directories_duration" qtlid="8218">Durée</translation>
+   <translation id="telephony_line_answer_voicemail_directories_main_FAQ">Ecoute et gestion des messages - Guide</translation>
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/voicemail/management/translations/Messages_fr_FR.xml]:
```suggestion
   <translation id="telephony_line_answer_voicemail_directories_main_FAQ">Écoute et gestion des messages - Guide</translation>
```

File: client/app/telecom/telephony/line/tones/telecom-telephony-line-tones.html [hunk 3]
```diff
@@ -9,8 +9,15 @@
 
     <div class="col-md-6">
         <div class="widget-presentation">
-            <h2 class="widget-presentation-title"
-                data-translate="telephony_line_tones_title">
+            <h2 class="widget-presentation-title">
+                {{:: 'telephony_line_tones_title' | translate }}
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/tones/telecom-telephony-line-tones.html]: Prefer using `ngBind` over `{{` (`data-translate`).

File: client/app/telecom/telephony/line/phone/order/translations/Messages_fr_FR.xml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <translations>
-   <translation id="telephony_line_phone_order_title" qtlid="336844">Commander un Plug &amp; Phone</translation>
-   <translation id="telephony_line_phone_order_change_title" qtlid="458858">Changer de téléphone Plug &amp; Phone</translation>
+   <translation id="telephony_line_phone_order_title" qtlid="336844">Commander un Plug & Phone</translation>
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/phone/order/translations/Messages_fr_FR.xml]: Could you please stay with `&amp;`? That's a requirements when using translations scripts.

File: client/app/telecom/telephony/line/phone/order/telecom-telephony-line-phone-order.html [hunk 5]
```diff
@@ -4,11 +4,25 @@
...
+            target="_blank"
+            rel="noopener"
+            data-ng-href="{{ PhoneOrderCtrl.links.telephony.phoneTroubleShooting }}">
+                <i class="fa fa-icon fa-external-link" aria-hidden="true"></i>
+            </a>
         </h1>
-        <h1 data-ng-if="PhoneOrderCtrl.phone"
-            data-translate="telephony_line_phone_order_change_title">
+        <h1 data-ng-if="PhoneOrderCtrl.phone">
+            {{:: 'telephony_line_phone_order_change_title' | translate }}
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/phone/order/telecom-telephony-line-phone-order.html]: Prefer using `ngBind` over `{{` (`data-translate`)

File: client/app/telecom/telephony/line/phone/order/telecom-telephony-line-phone-order.html [hunk 6]
```diff
@@ -4,11 +4,25 @@
         <tuc-section-back-link data-tuc-section-back-link-to-state="telecom.telephony.line.phone"
                            data-tuc-section-back-link-title="{{ 'telephony_group_line_phone_section_back_link' | translate }}">
         </tuc-section-back-link>
-        <h1 data-ng-if="!PhoneOrderCtrl.phone"
-            data-translate="telephony_line_phone_order_title">
+        <h1 data-ng-if="!PhoneOrderCtrl.phone">
+            {{:: 'telephony_line_phone_order_title' | translate }}
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/phone/order/telecom-telephony-line-phone-order.html]: Prefer using `ngBind` over `{{` (`data-translate`)

File: client/app/telecom/telephony/line/management/restrictions/telecom-telephony-line-management-restrictions.html [hunk 7]
```diff
@@ -4,7 +4,16 @@
         <tuc-section-back-link data-tuc-section-back-link-to-state="telecom.telephony.line"
                            data-tuc-section-back-link-title="{{:: 'telephony_group_line_section_back_link' | translate }}">
         </tuc-section-back-link>
-        <h1 data-translate="telephony_line_restrictions_title"></h1>
+        <h1>
+            {{:: 'telephony_line_restrictions_title' | translate }}
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/management/restrictions/telecom-telephony-line-management-restrictions.html]: Prefer using `ngBind` over `{{` (`data-translate`)

File: client/app/telecom/telephony/line/calls/timeCondition/telecom-telephony-line-calls-time-condition.html [hunk 8]
```diff
@@ -4,7 +4,16 @@
         <tuc-section-back-link data-tuc-section-back-link-to-state="telecom.telephony.line.calls"
                            data-tuc-section-back-link-title="{{ 'telephony_line_calls_back_link' | translate }}">
         </tuc-section-back-link>
-        <h1 data-translate="telephony_line_calls_time_condition_title"></h1>
+        <h1>
+            {{:: 'telephony_line_calls_time_condition_title' | translate }}
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/calls/timeCondition/telecom-telephony-line-calls-time-condition.html]: Prefer using `ngBind` over `{{` (`data-translate`)

File: client/app/telecom/telephony/line/calls/simultaneousLines/telecom-telephony-line-calls-simultaneousLines.html [hunk 9]
```diff
@@ -3,7 +3,16 @@
     <header>
         <tuc-section-back-link data-tuc-section-back-link-to-state="telecom.telephony.line.calls" data-tuc-section-back-link-title="{{ 'telephony_line_calls_back_link' | translate }}">
         </tuc-section-back-link>
-        <h1 data-translate="telephony_line_actions_line_calls_simultaneous_line_title"></h1>
+        <h1>
+            {{:: 'telephony_line_actions_line_calls_simultaneous_line_title' | translate }}
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/calls/simultaneousLines/telecom-telephony-line-calls-simultaneousLines.html]: Prefer using `ngBind` over `{{` (`data-translate`)

File: client/app/telecom/telephony/line/calls/forward/telecom-telephony-line-calls-forward.html [hunk 10]
```diff
@@ -4,7 +4,16 @@
         <tuc-section-back-link data-tuc-section-back-link-to-state="telecom.telephony.line.calls"
                            data-tuc-section-back-link-title="{{ 'telephony_line_calls_back_link' | translate }}">
         </tuc-section-back-link>
-        <h1 data-translate="telephony_line_actions_line_calls_forward_title"></h1>
+        <h1>
+            {{:: 'telephony_line_actions_line_calls_forward_title' | translate }}
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/calls/forward/telecom-telephony-line-calls-forward.html]: Prefer using `ngBind` over `{{` (`data-translate`)

File: client/app/telecom/telephony/line/calls/displayNumber/telecom-telephony-line-calls-displayNumber.html [hunk 11]
```diff
@@ -4,7 +4,16 @@
         <tuc-section-back-link data-tuc-section-back-link-to-state="telecom.telephony.line.calls"
                            data-tuc-section-back-link-title="{{ 'telephony_line_calls_back_link' | translate }}">
         </tuc-section-back-link>
-        <h1 data-translate="telephony_line_actions_line_calls_display_number_title"></h1>
+        <h1>
+            {{:: 'telephony_line_actions_line_calls_display_number_title' | translate }}
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/calls/displayNumber/telecom-telephony-line-calls-displayNumber.html]: Prefer using `ngBind` over `{{` (`data-translate`)

File: client/app/telecom/telephony/line/calls/click2Call/telecom-telephony-line-calls-click2Call.html [hunk 12]
```diff
@@ -4,7 +4,16 @@
         <tuc-section-back-link data-tuc-section-back-link-to-state="telecom.telephony.line.calls"
                            data-tuc-section-back-link-title="{{ 'telephony_line_calls_back_link' | translate }}">
         </tuc-section-back-link>
-        <h1 data-translate="telephony_group_line_calls_click2call_title"></h1>
+        <h1>
+            {{:: 'telephony_group_line_calls_click2call_title' | translate }}
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/calls/click2Call/telecom-telephony-line-calls-click2Call.html]: Prefer using `ngBind` over `{{` (`data-translate`)

File: client/app/telecom/telephony/line/calls/callWaiting/telecom-telephony-line-calls-callWaiting.html [hunk 13]
```diff
@@ -4,7 +4,16 @@
         <tuc-section-back-link data-tuc-section-back-link-to-state="telecom.telephony.line.calls"
             data-tuc-section-back-link-title="{{ 'telephony_line_calls_back_link' | translate }}">
         </tuc-section-back-link>
-        <h1 data-translate="telephony_line_actions_line_calls_call_waiting_title"></h1>
+        <h1>
+            {{:: 'telephony_line_actions_line_calls_call_waiting_title' | translate }}
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/calls/callWaiting/telecom-telephony-line-calls-callWaiting.html]: Prefer using `ngBind` over `{{` (`data-translate`)

File: client/app/telecom/telephony/line/answer/voicemailManagement/telecom-telephony-line-answer-voicemailManagement.html [hunk 14]
```diff
@@ -4,7 +4,16 @@
         <tuc-section-back-link data-tuc-section-back-link-to-state="telecom.telephony.line.answer"
                            data-tuc-section-back-link-title="{{ 'telephony_line_answer_back_link' | translate }}">
         </tuc-section-back-link>
-        <h1 data-translate="telephony_line_answer_voicemail_directories_main_title"></h1>
+        <h1>
+            {{:: 'telephony_line_answer_voicemail_directories_main_title' | translate }}
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/answer/voicemailManagement/telecom-telephony-line-answer-voicemailManagement.html]: Prefer using `ngBind` over `{{` (`data-translate`)

File: client/app/telecom/telephony/billingAccount/phonebook/telecom-telephony-billing-account-phonebook.html [hunk 15]
```diff
@@ -165,9 +163,16 @@
                 </section><!-- .widget-presentation -->
                 <section class="widget-presentation">
                     <header class="widget-presentation-header">
-                        <h2 class="widget-presentation-title"
-                            data-translate="telephony_phonebook_contact_title">
-                        </h2>
+                        <h2 class="widget-presentation-title">
+                            {{:: 'telephony_phonebook_contact_title' | translate }}
```
Comment (antleblanc) [inline client/app/telecom/telephony/billingAccount/phonebook/telecom-telephony-billing-account-phonebook.html]: Prefer using `ngBind` over `{{` (`data-translate`)

File: client/app/telecom/telephony/billingAccount/phonebook/telecom-telephony-billing-account-phonebook.html [hunk 16]
```diff
@@ -165,9 +163,16 @@
...
+                            <a class="oui-button oui-button_link oui-button_icon-right"
+                            title="{{ ::'telephony_phonebook_contact_FAQ' | translate }} {{ ::'common_newtab' | translate }}"
+                            target="_blank"
+                            rel="noopener"
+                            data-ng-href="{{ PhonebookCtrl.links.telephony.contactBook }}">
+                                <i class="fa fa-icon fa-external-link" aria-hidden="true"></i>
+                            </a>
+                            </h2>
```
Comment (antleblanc) [inline client/app/telecom/telephony/billingAccount/phonebook/telecom-telephony-billing-account-phonebook.html]: Indentation issue.

File: client/app/telecom/telephony/billingAccount/abbreviatedNumbers/telecom-telephony-abbreviatedNumbers.html [hunk 17]
```diff
@@ -6,8 +6,15 @@
 
     <section class="widget-presentation"
         data-ng-hide="AbbreviatedNumbersCtrl.loading">
-        <h2 class="widget-presentation-title"
-            data-translate="telephony_abbreviated_numbers_title">
+        <h2 class="widget-presentation-title">
+            {{:: 'telephony_abbreviated_numbers_title' | translate }}
```
Comment (antleblanc) [inline client/app/telecom/telephony/billingAccount/abbreviatedNumbers/telecom-telephony-abbreviatedNumbers.html]: Prefer using `ngBind` over `{{` (`data-translate`)

File: client/app/telecom/telephony/line/calls/callWaiting/telecom-telephony-line-calls-callWaiting.html (hunk 13 above)
Comment (antleblanc) [reply client/app/telecom/telephony/line/calls/callWaiting/telecom-telephony-line-calls-callWaiting.html]: I'm using directly the `translate` directive.

```html
<span data-translate="telephony_line_actions_line_calls_call_waiting_title"></span>
```

Comment (Steffy29) [reply client/app/telecom/telephony/line/calls/callWaiting/telecom-telephony-line-calls-callWaiting.html]: Thanks for the explanation, I wasn't sure.

**Comment by Steffy29** [comment]
Need to the refactor the code to apply model Guidelines to add guides links

**Pull request closed without merging**